cmd/harmony/config_migrations.go

```diff
@@ -188,6 +189,9 @@ func init() {
 		if confTree.Get("P2P.DiscConcurrency") == nil {
 			confTree.Set("P2P.DiscConcurrency", defaultConfig.P2P.DiscConcurrency)
 		}
+		if confTree.Get("P2P.MaxConnsPerIP") == nil {
```

Comment:
You should create a new version when adding a new field. Otherwise, the field will not be updated.